tests/testthat/test-summary-log.R

```diff
@@ -0,0 +1,100 @@
+context("Test creating summary logs")
+
+if (Sys.getenv("METWORX_VERSION") == "" && Sys.getenv("DRONE") != "true") {
+  skip("test-summary only runs on Metworx or Drone")
+}
+
+source("data/test-workflow-ref.R")
```

Comment:
We should change from the `source()` pattern to the `testthat` handling for special files, as described in metrumresearchgroup/mrgvalidatetestreference#8 (at least for all new development, if we don't want to change everything right now).